src/openloco/town.h

```diff
@@ -40,7 +40,15 @@ namespace openloco
         int16_t company_ratings[15];    // 0x3A
         uint16_t companies_with_rating; // 0x58
         town_size size;                 // 0x5A
-        uint8_t pad_5B[0x1A6 - 0x5B];
+        uint8_t history_size;           // 0x5B (<= 20 * 12)
+        uint8_t history[20 * 12];       // 0x5C (20 years, 12 months)
+        int32_t history_min_population; // 0x14C
+        uint8_t pad_150[0x158 - 0x150];
+        int16_t unk_158[0x20];
```

Comment:
I think these are related to cargo deliveries since the last update. This implies more cargo means increased build speed.